lib/auditor.rb

```diff
@@ -10,6 +10,7 @@ def initialize
 
   def audit(dir:, repo:)
     Dir.chdir(dir) do
+      out, status = Open3.capture2 'bundle'
```

Comment:
Note that "install" is the default bundler thing to do.  I can add "install" or a comment to clarify if you like.